services/galley/src/Galley/API/Update.hs

```diff
@@ -394,6 +426,22 @@ data CheckedOtrRecipients
     | InvalidOtrSenderClient
         -- ^ Invalid sender (client).
 
+withValidOtrBroadcastRecipients
+    :: UserId
+    -> ClientId
+    -> OtrRecipients
+    -> OtrFilterMissing
+    -> UTCTime
+    -> ([(Member, ClientId, Text)] -> Galley ())
+    -> Galley Response
+withValidOtrBroadcastRecipients usr clt rcps val now go = Teams.withBindingTeam usr $ \tid -> do
+    tMembers <- fmap (view userId) <$> Data.teamMembers tid
+    contacts <- getContactList usr
+    let users = tMembers ++ contacts
```

Comment:
since `tMembers` and `contacts` may overlap, maybe better to remove potential duplicates in `users` ?